src/psyclone/tests/domain/lfric/lfric_stencil_tests.py

```diff
@@ -789,7 +789,7 @@ def test_single_stencil_cross2d(dist_mem, tmpdir):
 
     '''
     _, invoke_info = parse(
-        os.path.join(BASE_PATH, "19.25_single_stencil_cross2d.f90"),
+        os.path.join(BASE_PATH, "19.26_single_stencil_cross2d.f90"),
```

Comment:
The number of the file was duplicated with another at some point since it went on so i changed mine to avoid duplicate numbering. The actual filename change can be seen at the bottom of the files changed list.

Reply:
Thanks for the explanation!